packages/terra-table/src/subcomponents/ColumnResizeHandle.jsx

```diff
@@ -108,6 +108,8 @@ const ColumnResizeHandle = (props) => {
 
   // Ref variable for native resize handle element
   const resizeHandleRef = useRef();
+  // Ref variable to prevent active state on mouse drag
+  const handleFocus = useRef(true);
```

Comment:
Can't we use the isActive prop?  It seems like that would work without this extra logic.

Reply:
In that scenario, we don't have the isActive prop that we can leverage.  Also, focus is being given to the DataGrid externally.  Why is the isActive prop not sufficient would be my question.